Assets/Scripts/Manager/levelManager.cs

```diff
@@ -102,17 +102,24 @@ bool isGhostMode ()
 		Camera.main.GetComponent<CameraFollow> ().findPlayer ();
 	}
 
+	void moveCamera(Vector3 position){
+		Camera.main.GetComponent<CameraFollow> ().ghostModeSelect (position);
+	}
+
 	// selecting player from the array 'dolls'
 	void selectPlayer(){
 		highlight(dolls[Mathf.Abs(dollIndex)]);
+		moveCamera (dolls [Mathf.Abs (dollIndex)].transform.position);
```

Comment:
Is it necessary to do get the absolute value of the doll index? Is there a situation where the doll index is negative?